### PR TITLE
Change to customizable threshold value for termination

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Changed
```
Comment (fiunchinho) [inline CHANGELOG.md]:
```suggestion
### Changed
```

File: helm/node-problem-detector/values.yaml [hunk 2]
```diff
@@ -4,7 +4,9 @@ node-problem-detector:
 
   labels:
     application.giantswarm.io/team: phoenix
-
+  
+  used_space_threshold: 99
```
Comment (fiunchinho) [inline helm/node-problem-detector/values.yaml]: Let's use camel case as suggested [in the docs](https://helm.sh/docs/chart_best_practices/values/#naming-conventions).

Comment (T-Kukawka) [reply helm/node-problem-detector/values.yaml]: done

File: helm/node-problem-detector/templates/configmap.yaml [hunk 3]
```diff
@@ -39,7 +39,7 @@ data:
     used_space=$(df "$disk_path" --output=pcent | tail -n1|sed 's/%//')
 
     # Check if the disk is full
-    if [ $used_space -ge 99 ]; then
+    if [ $used_space -ge { node-problem-detector.usedSpaceThreshold }} ]; then
```
Comment (fiunchinho) [inline helm/node-problem-detector/templates/configmap.yaml]:
```suggestion
    if [ $used_space -ge {{ node-problem-detector.usedSpaceThreshold }} ]; then
```

**Comment by fiunchinho** [comment]
I'd test it before merging just in case.

**Comment by T-Kukawka** [comment]
i definitely want to test this @fiunchinho :D

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**